diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -104,17 +106,90 @@ impl ToSql<Timestamp, Sqlite> for NaiveDateTime {
     }
 }
 
+impl FromSql<TimestamptzSqlite, Sqlite> for NaiveDateTime {
```

Comment:
We should mark these impl with
`#[cfg(all(feature = "sqlite", feature = "chrono")]` so that the corresponding features show up in the API documentation.

(This applies to all other `FromSql`/`ToSql` impls added in this file)